src/app.pcss

```
@@ -94,6 +94,7 @@ body {
 	background-color: var(--colors-ultra-low);
 	color: var(--colors-ultra-high);
 	font-family: var(--font-family-sans-serif);
+	height: 100%;
```

I think setting it to `vh` is safer

```suggestion
	height: vh;
```

---

most likely we also want `width: 100vw`

---

When I set body width to 100vw, a problem appeared when there was vertical scrolling on the page, so the width expanded to include the scroll panel and with that, horizontal scrolling also appeared.